.gitlab-ci.yml

```diff
@@ -3016,6 +3016,29 @@ pupernetes-tags-7:
   - VERSION=$(inv -e agent.version --major-version 7)
   - inv -e e2e-tests --image=datadog/agent:${VERSION}
 
+notify-on-success:
+  extends: .slack-notifier-base
+  before_script: ["# noop"]
```

Comment:
No longer needed once we merge https://github.com/DataDog/datadog-agent/pull/5083